src/index.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import router from './routes/routes';
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
You need only `app.use(bodyParser.json());` - urlencoded is used for multipart form data objects

Reply:
So, are you sending the query JSON in the headers with the Content-Type as application/json? I can't get anything to show up in the req.body if I change that to bodyParser.json() and add it in headers. Noted on the use of urlencoded.

Reply:
This took an embarrassingly long amount of time to work out how to send this correctly in Postman. Ah man.